package.json

```diff
@@ -85,6 +85,7 @@
     "morgan": "1.2.0",
     "ms": "0.7.1",
     "node-sass": "3.7.0",
+		"notifications-panel": "git+https://github.com/Automattic/notifications-panel.git",
```

Comment:
👍 
we'll pull from `master` for the moment, but I personally think it'll be best to use a release tag.